admin-ui/app/types/react-helmet.d.ts

```diff
@@ -0,0 +1,20 @@
+declare module 'react-helmet' {
+  import { ComponentType, HelmetData } from 'react-helmet'
```

Comment:
Fix circular import and incorrect type source.

This line has a critical circular import issue: it imports from the same module being declared (`'react-helmet'`). Additionally, `ComponentType` is a React type, not a react-helmet export.



Apply this diff to correct the imports:

```diff
-  import { ComponentType, HelmetData } from 'react-helmet'
+  import { ComponentType, ReactNode } from 'react'
```

Note: If `HelmetData` is required as an export, it should be defined as an interface within this declaration module rather than imported.